app/controllers/activities_controller.rb

```diff
@@ -5,15 +5,34 @@ def index
   end
 
   def new
+    @activity = Activity.new
+  end
 
+  def create
+    @activity = Activity.new(activity_params)
+
+    respond_to do |format|
+      if @activity.save
+        format.html { redirect_to @activity, notice: 'Activity was successfully created.' }
+        format.json { render action: 'show', status: :created, location: @activity }
```

Comment:
we don't need html format now. More than that, I would recommend to switch to `respond_with`